README.md

```diff
@@ -43,13 +43,19 @@ curl -L https://foundry.paradigm.xyz | sh
 git submodule update --init
 ```
 
-3. Run tests:
+3. Update foundry to get the baseline install:
```

Comment:
isn't the shell script spinning up the last version already? Haven't checked tbh

Reply:
instructions in the readme didn't work for me until `foundryup` was run once